pkg/apis/extensions/types.go

```diff
@@ -35,6 +35,15 @@ import (
 	"k8s.io/kubernetes/pkg/util/intstr"
 )
 
+// The following identify resource constants for Kubernetes object types
+const (
+	// ReplicaSets, number
+	ResourceReplicaSets api.ResourceName = "replicasets"
```

Comment:
@derekwaynecarr seems like now is the time to establish the pattern.  Based on the validator, I think I'd say `APIGroupName/resourceName`.